pkg/server/handlers_components.go

```diff
@@ -424,6 +425,17 @@ func (g *globalHandler) getMetrics(c *gin.Context) {
 		metricsSince = now.Add(-dur)
 	}
 
+	if enableGlobalMetricsStore := c.Query("enableGlobalMetricsStore"); enableGlobalMetricsStore == "true" {
```

Comment:
It's unreasonable that enable features based on query param, how this is implemented is a gpud internal detail and should not be perceived externally. We should add a feature gate or flag to control whether enable global metric store feature..